cmake/unit_testing.cmake

```diff
@@ -29,4 +29,6 @@ add_subdirectory( src/ACEtk/interpretation/MP1/test )
 #######################################################################
 # python tests 
 #######################################################################
-add_subdirectory( python/test )
+if( ACEtk.python )
+  add_subdirectory( python/test )
```

Comment:
I was a bit confused as to how this works, but it looks like you added a CMakeLists.txt into the Python folder separate from this PR.  I like having this available from CTest for sure.